front/src/components/boxs/device-in-room/device-features/CurrentSetpointDeviceFeature.jsx

```diff
@@ -0,0 +1,107 @@
+import get from 'get-value';
```

Comment:
This component is exactly the same as the ThermostatDeviceFeature component, why clone it and not just make it work for both cases?

Reply:
Indeed ^^ In this case, it was the fear of doing something wrong by modifying an existing component that ultimately made me create one on the side ^^
I'm going for a renaming to Setpoint